components/feature/autofill/src/main/java/mozilla/components/feature/autofill/response/fill/LoginFillResponseBuilder.kt

```diff
@@ -55,6 +56,26 @@ internal data class LoginFillResponseBuilder(
             searchDatasetBuilder.build(context, configuration, imeSpec)
         )
 
+        builder.handleSaveInfo(parsedStructure)
+
         return builder.build()
     }
 }
+
+@RequiresApi(Build.VERSION_CODES.O)
+internal fun FillResponse.Builder.handleSaveInfo(ps: ParsedStructure) {
```

Comment:
nit: I feel like abbreviations like `ps` make it hard to read code since you always have to keep the mapping in mind. Maybe we can just name it `structure`?

Reply:
Sure!